app/assets/javascripts/hydra-editor/manage_repeating_fields.js

```diff
@@ -3,139 +3,151 @@
 // These assumptions are reflected in the MultiValueInput class.
 
 var HydraEditor = (function($) {
-      var FieldManager = function (element, options) {
-          this.element = $(element);
-          this.options = options;
-
-          this.controls = $(options.controlsHtml);
-          this.remover  = $(options.removeButtonHtml);
-          this.adder    = $(options.addButtonHtml);
-
-          this.fieldWrapperClass = options.fieldWrapperClass;
-          this.warningClass = options.warningClass;
-          this.listClass = options.listClass;
-
-          this.init();
-      }
-
-      FieldManager.prototype = {
-          init: function () {
-              this._addInitialClasses();
-              this._appendControls();
-              this._attachEvents();
-              this._addCallbacks();
-          },
-
-          _addInitialClasses: function () {
-              this.element.addClass("managed");
-              $(this.fieldWrapperClass, this.element).addClass("input-group input-append");
-          },
-
-          _appendControls: function() {
-              $(this.fieldWrapperClass, this.element).append(this.controls);
-              $(this.fieldWrapperClass+':not(:last-child) .field-controls', this.element).append(this.remover);
-              $('.field-controls:last', this.element).append(this.adder);
-          },
-
-          _attachEvents: function() {
-              var _this = this;
-              this.element.on('click', '.remove', function (e) {
-                _this.removeFromList(e);
-              });
-              this.element.on('click', '.add', function (e) {
-                _this.addToList(e);
-              });
-          },
-
-          _addCallbacks: function() {
-              this.element.bind('managed_field:add', this.options.add);
-              this.element.bind('managed_field:remove', this.options.remove);
-          },
-
-          addToList: function( event ) {
+    var FieldManager = function (element, options) {
+        this.element = $(element);
+        this.options = options;
+
+        this.options.label = '';
+        var $label = this.element.find("label");
+
+        if ( $label.size ) {
+          var label = $label.data('label') || $.trim($label.contents().filter(function() { return this.nodeType === 3; }).text());
+          this.options.label = ' ' + label;
+        }
+
+        this.controls = $("<span class=\"input-group-btn field-controls\">");
```

Comment:
This should be passed in from the options.  Did you start from an out of date commit?